core/src/com/projectkorra/projectkorra/ability/CoreAbility.java

```diff
@@ -338,11 +329,9 @@ public static void removeAll() {
 	 * @return a CoreAbility instance or null
 	 */
 	public static <T extends CoreAbility> T getAbility(final Player player, final Class<T> clazz) {
-		final Collection<T> abils = getAbilities(player, clazz);
-		if (abils.iterator().hasNext()) {
-			return abils.iterator().next();
-		}
-		return null;
+		final Collection<T> abilities = getAbilities(player, clazz);
+		final Iterator<T> iterator = abilities.iterator();
+		return iterator.hasNext() ? iterator.next() : null;
```

Comment:
Why?

Reply:
I thought it was cleaner but if you also disagree I can revert it. It's done because calling iterator twice is redundant and because the return is a really easy ternary assuming you have the iterator